checkov/secrets/git_history_store.py

```diff
@@ -10,6 +10,20 @@
 if TYPE_CHECKING:
     from detect_secrets.core.potential_secret import PotentialSecret
 
+RENAME_STR = 'rename'
+FILE_RESULTS_STR = 'file_results'
+RAW_STORE_TYPES = {RENAME_STR, FILE_RESULTS_STR}
+
+
+class RawStore(TypedDict):
+    file_results: List[PotentialSecret]
+    file_name: str
+    commit: Dict[str, str | Dict[str, str]]
+    commit_hash: str
+    type: str  # rename / file results
```

Comment:
if you want to make it very precise, then you can leverage `Literal`
```python
from typing_extensions import Literal

type:  Literal[RENAME_STR, FILE_RESULTS_STR]
```
not 100% sure, if you can use the variables directly or have to write is=t as a string again, but also need to type hint the consts too.
```python
RENAME_STR: "Literal['rename']" = 'rename'
```

Reply:
for this super simple thing seems like an overkill
since I will probably edit it later I will do it in the PR if it will be relevant :) 
thanks @gruebel